golang/cosmos/types/ibc_packet_test.go

```diff
@@ -0,0 +1,119 @@
+package types_test
+
+import (
+	"bytes"
+	"encoding/json"
+	fmt "fmt"
+	"reflect"
+	"testing"
+
+	"github.com/Agoric/agoric-sdk/golang/cosmos/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	clienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
+	ibcexported "github.com/cosmos/ibc-go/v6/modules/core/exported"
+)
+
+func NewChannelPacket() channeltypes.Packet {
+	return channeltypes.NewPacket(
+		[]byte("data"),
+		987654321098765432,
+		"port-src", "channel-13",
+		"port-dst", "channel-42",
+		clienttypes.Height{},
+		123456789012345678,
+	)
+}
+
+func TestPacket(t *testing.T) {
+	testCases := []struct {
+		name   string
+		packet ibcexported.PacketI
+	}{
+		{
+			name:   "channeltypes.Packet",
+			packet: NewChannelPacket(),
+		},
+		{
+			name:   "types.Packet",
+			packet: types.CopyToIBCPacket(NewChannelPacket()),
+		},
```

Comment:
"types" doesn't really mean anything in output from this file, and "channeltypes" isn't much better. Suggestion (here and for other `Test…` functions):
```suggestion
		{
			name:   "ibc-go channel Packet",
			packet: NewChannelPacket(),
		},
		{
			name:   "agoric-sdk IBCPacket",
			packet: types.CopyToIBCPacket(NewChannelPacket()),
		},
```

Reply:
👍